requirements.txt

```diff
@@ -1,5 +1,5 @@
 boto
-Django==1.4
+Django==1.11.12
 django-widget-tweaks==1.3
```

Comment:
Should we consider bumping the django-widget-tweaks version here as well?  Current version is 1.4.2 on Github